CONTRIBUTING.md

```diff
@@ -8,7 +8,13 @@ The front end of MyLA is written in JavaScript using [React](https://reactjs.org
 The code for the front-end is located under `/assets/src/`. MyLA uses functional components instead of class-based components and uses [React hooks](https://reactjs.org/docs/hooks-intro.html) for state and side-effects.
 
 ### Back-end
-To Do
+MyLA Backend is build using [Django](https://www.djangoproject.com/) Framework, [MySQL](https://www.mysql.com/) DataBase,
+Cron schedule for getting the Canvas context and event data to the MySQL DB each institution might set this up differently based on the infrastruture support.
+MyLA can be run as a stand alone tool with [SAML](https://developers.onelogin.com/saml) support and as LTI tool. 
+
+More info of various institutions infrastructure set up is [here](https://github.com/tl-its-umich-edu/my-learning-analytics/wiki/Myla-institutions-Architecture-flow) 
```

Comment:
Adding backend tech stack, I also created a Wiki page for adding institutions architectural diagram, I have added Michigan design flow created by @jonespm there. Hope we can see other institutions sharing there as well.